R/polars_options.R

```diff
@@ -104,17 +111,17 @@ print.polars_options <- function(x, ...) {
 #' @param is_missing Is `x` missing in the calling function?
 #' @param default The default of `x` in the calling function
 #' @noRd
-use_option_if_missing <- function(x, is_missing, default) {
+use_option_if_missing <- function(x, is_missing, default, prefix = "polars.") {
```

Comment:
I don't understand this change, is there any option we don't want to prefix with `polars.`? This adds noise in `series__to_r_vector`. We could just define `prefix <- "polars.to_r_vector."` at the top of this function.